### PR TITLE
Requesting a pull to mozilla:v1 from k88hudson:data

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,8 @@
     "react": "0.13.1",
     "react-hammerjs": "0.2.2",
     "react-imageloader": "1.2.0",
+    "react-loadermixin": "1.0.4",
```
Comment (thisandagain) [inline package.json]: This can be removed ... looks like a conflicting change with #1662

Comment (k88hudson) [inline package.json]: ah, ok. showed up in my rebase conflict.

File: www_src/lib/api-fake-data.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+module.exports = {
```
Comment (thisandagain) [inline www_src/lib/api-fake-data.js]: Mind if we call this `mock.js` or `mock.json` and keep it isolated to the project (map) page?

File: www_src/lib/api-mock.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      cb(null, {});
+    }
+  }
+
+  else {
+    cb(new Error('That uri does not exist'));
+  }
+
+  // Update Android
+  if (window.Android && ['put', 'post', 'delete'].indexOf(method) > -1) {
```
Comment (thisandagain) [inline www_src/lib/api-mock.js]: Good catch. I should add this to the actual API module as well. :+1:

Comment (k88hudson) [inline www_src/lib/api-mock.js]: maybe, but in the case of actual data, there's definitely a possibility that a GET response differs from the currently cached one. if that's what you mean.

Comment (thisandagain) [inline www_src/lib/api-mock.js]: I'm just thinking that we should *never* cache POST, PUT, or DELETE regardless of the `useCache` flag.

**Comment by thisandagain** [comment]
Good start. R+ :poodle: :boom:

File: www_src/lib/api-mock.js (hunk 3 above)
Comment (k88hudson) [inline www_src/lib/api-mock.js]: ah! right. good call

**Pull request merged**